.github/workflows/pr_labeler_validator.yaml

```
@@ -0,0 +1,152 @@
+                        await github.rest.issues.createComment({
+                          owner: context.repo.owner,
+                          repo: context.repo.repo,
+                          issue_number: context.payload.pull_request.number,
+                          body: message
+                        });
+                      }
+
+                      core.setFailed("Branch Naming Convention 위반");
+
+            - name: Check PR title format
+              run: |
```

actionlint (1.7.7)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)